packages/bpk-component-chip-group/src/BpkChipGroupSingleSelect.tsx

```diff
@@ -20,12 +20,15 @@ import { useState } from 'react';
 
 import BpkChipGroup, { type ChipItem, type SingleSelectChipItem, type ChipGroupProps } from './BpkChipGroup';
 
-export type SingleSelectProps = {
+type CommonSingleSelectProps = {
```

Comment:
Extract common base to extend rather than using Omit below to keep TS happy with differences